cluster/store_test.go

```diff
@@ -3,11 +3,85 @@ package cluster
 import (
 	"context"
 	"github.com/stretchr/testify/require"
+	"go.etcd.io/etcd/clientv3"
 	"testing"
+	"time"
 )
 
 func TestNewKVStore(t *testing.T) {
 	store, err := NewKVStore(context.Background(), "")
 	require.NoError(t, err)
 	require.NotNil(t, store)
 }
+
+func (suite *EtcdDependentSuite) TestKVGet() {
+	suite.SetupTest()
```

Comment:
I don't think we have any setup for the tests, only for the suite

Reply:
I changed that func to also clear the store prefixed data in etcd.